moveit_core/collision_detection/src/collision_octomap_filter.cpp

```diff
@@ -187,6 +186,7 @@ bool getMetaballSurfaceProperties(const octomap::point3d_list& cloud, const doub
   else  // just get normals, no depth
   {
     octomath::Vector3 gradient;
+    double intensity;
```

Comment:
It's strange that this variable doesn't need to be initialized but I checked the implementation of `sampleCloud()` and indeed, it does not need to be initialized.

Let's do it anyway, to be on the safe side.

```suggestion
    double intensity = 0;
```